main.go

```diff
@@ -192,6 +192,13 @@ func runSuite(suite TestSuite) []TestResult {
 			}
 		}
 
+		unused := vars.Unused()
+		if len(unused) != 0 {
+			traces := result.Traces
+			lastTrace := traces[len(traces)-1]
+			lastTrace.addError(fmt.Errorf("Unused variables found: %s", unused))
```

Comment:
Variables is internal name. In a suite it's arguments ("args" property). I think it will be better to use this word. 
"There are declared/remembered arguments that were not used: argName1, argName2"